### DISCUSSION TITLE
Pagination vs load more and infinite scroll

### DISCUSSION THREAD

**Discussion by benalleng** [btcpayserver/btcpayserver#4834] [Ideas / Feature Requests]
According to some discussion on Mattermost @NicolasDorier and @dstrukt had mentioned
> @benalleng actually I'd like to remove all pagination and way to skip deep.
It would be only me it should only be a See more button.
main reason is that paging is super bad for database. This is why you never see it in facebook,youtube and reddit or any service that have lot's of data.
if people wants more then we should have better search
or a good way to export lots of data in CSV
I don't think we should even try to have pages

> we had talked about removing pagination in favor of something along those lines @nicolasdorier, but i believe @kukks had some thoughts and considerations on this approach IIRC as well

There is a lot of scattered discussion from #3954 resolved mostly by #4074 as well as #3835

currently pagination exists in 
- Server Settings > Logs
- Invoices
- Payment Requests
- Pull Payments
- Payouts

Three paths forward to consider
1. we completely removing the pagination component in favor of a `Load more` button that transitions into infinite scrolling for all components
2. We choose which components should have pagination and which ones should scroll
3. No changes necessary pagination is fine where it is

**Comment by dstrukt** [comment]
cc @Kukks 

Will try to draft up some thoughts this evening - thanks for making this a discussion!
